app/src/main/java/ch/epfl/swissteam/services/SettingsContract.java

```diff
@@ -0,0 +1,35 @@
+package ch.epfl.swissteam.services;
+
+import android.provider.BaseColumns;
+
+/**
+ * Contract defining the database table name and column names
+ * for a single table representing the settings of a client
+ *
+ * @author Julie Giunta
+ */
+public class SettingsContract {
+    public static final String LOCAL_DB_NAME = "ch.epfl.swissteam.services.localDB";
+    public static final int DB_VERSION = 1;
+
+    private SettingsContract(){}
+
+    public static class SettingsEntry implements BaseColumns{
+        public static final String TABLE_NAME = "entry";
```

Comment:
maybe the name should be "settings_entry", if we choose to add some table ( :') ) to keep it clear